### PR TITLE
Password Generator Using React.js

### PR DESCRIPTION
**Pull request by PranavBarthwal**
# Pull Request for WebMasterLog 💡

## Issue Title: Password Generator Using React.js

- **Name of the project** :  Password-Generator
- **Category:**  React.js
- **Name:**  Pranav 
- **GitHub ID:**  https://github.com/PranavBarthwal
- **Idenitfy yourself:**  JWOC Contributor

Closes: #85 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
added random password generator project using react.

## Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made
by running on my local system

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

Screenshot 2024-02-14 235046

### CODE REVIEW

File: React-JS-Projects/Password-Generator/README.md
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Password-Generator/README.md:1]: Change the title and use the newly updated README Template of the repository.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@PranavBarthwal Keep this project inside a Basic named directory as the structure of `Vanilla-JS-Projects`

File: React-JS-Projects/Password-Generator/README.md
Comment (PranavBarthwal) [reply React-JS-Projects/Password-Generator/README.md:1]: okay

Comment (PranavBarthwal) [reply React-JS-Projects/Password-Generator/README.md:1]: Done

File: React-JS-Projects/Password-Generator/.eslintrc.cjs
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Password-Generator/.eslintrc.cjs:1]: delete this folder

File: React-JS-Projects/Basic/Password-Generator/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<h1 align='center'><b>💥 CosmoXplore : APoD 💥</b></h1>
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Basic/Password-Generator/README.md:1]: still readme is not updated!
go through readme template and apply here and change the title ok

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@PranavBarthwal  Delete the project directory outside the 'basic' folder.

**Pull request closed without merging**